x-pack/test/functional/apps/graph/feature_controls/graph_spaces.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from 'expect.js';
+import { SpacesService } from 'x-pack/test/common/services';
+import { KibanaFunctionalTestDefaultProviders } from '../../../../types/providers';
+
+// tslint:disable:no-default-export
+export default function({ getPageObjects, getService }: KibanaFunctionalTestDefaultProviders) {
+  const esArchiver = getService('esArchiver');
+  const spacesService: SpacesService = getService('spaces');
+  const PageObjects = getPageObjects(['common', 'graph', 'security', 'error']);
+  const testSubjects = getService('testSubjects');
+  const appsMenu = getService('appsMenu');
+
+  describe('spaces', () => {
+    before(async () => {
+      await esArchiver.load('empty_kibana');
+    });
+    describe('space with no features disabled', () => {
+      before(async () => {
+        await spacesService.create({
+          id: 'custom_space',
+          name: 'custom_space',
+          disabledFeatures: [],
+        });
+      });
+
+      after(async () => {
+        await spacesService.delete('custom_space');
+      });
+
+      it('shows graph navlink', async () => {
+        await PageObjects.common.navigateToApp('home', {
+          basePath: '/s/custom_space',
+        });
+        const navLinks = (await appsMenu.readLinks()).map(
+          (link: Record<string, string>) => link.text
+        );
+        expect(navLinks).to.contain('Graph');
+      });
+
+      it('shows save button', async () => {
```

Comment:
Can we add a test here for the delete button?

Reply:
Yup, good catch.